artifacts/deployment/etcdproxy-controller.yaml

```diff
@@ -0,0 +1,72 @@
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: kube-apiserver-storage
+spec:
+---
+kind: ClusterRole
```

Comment:
ok, i see the problem.  You need a clusterrole to see the etcdstorage.  Then you need a namespaced role for managing the things the controller needs in your namespace.

Reply:
Done